### ISSUE TITLE
Ubuntu Installation failure: The certificate is not trusted 

### ISSUE THREAD

**Issue by iusetabs** [cri-o/cri-o#5375]
**Description**

Cri-o cannot be installed securely due to an expired certificate on the https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/ repository. Also on the 'https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_18.04  repository.

**Steps to reproduce the issue:**
1. Follow the installation instructions for Ubuntu/Debian using Ubuntu 20,04 . 18.04 ( https://cri-o.io/ )

**Describe the results you received:**

- When executing apt-get update, the following error is returned
`Err:7 https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04  Release
  Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification.`
`N: Updating from such a repository can't be done securely, and is therefore disabled by default.`

- The final result is 
`E: Package 'cri-o-runc' has no installation candidate`

**Describe the results you expected:**
Of course to find the installation package. 

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `crio --version`:**

```
All Cri-o versions on Ubuntu
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
- Occurs whilst trying to install via a EC-2 Ubuntu image ( on both 18.04 and 20.04 ).

**Comment by iusetabs** [comment]
Run `apt-get upgrade` and then `apt-get update` solves the issue 🙈 Maybe helps somebody if they get stuck with the same!

**Comment by iusetabs** [comment]
On second thought, might be better to leave this open as installation should work out of the box ( i.e. it's not nice to have to run apt-get upgrade ), but not sure if this is on the AWS side or the Cri-o side. Please close if this is on the AWS side 😄

**Comment by haircommander** [comment]
this may actually be a documentation bug. Maybe after https://github.com/cri-o/cri-o/pull/5374 merges you can submit a PR that better documents what needs to happen? (or you can comment on it and @thanos1983 can apply the changes)

**Comment by thanos1983** [comment]
> this may actually be a documentation bug. Maybe after #5374 merges you can submit a PR that better documents what needs to happen? (or you can comment on it and @thanos1983 can apply the changes)

To be honest I am a bit also confused @haircommander.

As we saw on #5368 the problem was resolved by running `sudo apt-get update && `sudo apt-get install ca-certificates`. On my case on fresh clean installation as @iusetabs has noticed it requires (maybe without the CA) to run `sudo apt-get upgrade`.

I am not sure if on fresh image pull 20.04 to run upgrade so we can install CRI-O socket. It looks like overkill. All these problems where introduced after the CA update a few days ago.

I do not think that this is an error on CRI-O side it seems to be an error on Ubuntu side. They seem to be missing a CA that is getting pulled when we run upgrade. I think a new release from Ubuntu side should fix this issue.

Temporary solution is upgrade after installing CAs.

**Comment by haircommander** [comment]
does that workaround work for you @iusetabs ?

**Comment by justabaka** [comment]
That is a a few months old local CA certificates issue, not CRI-O's in any part. Your system administrator should've fixed this a long time ago. https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

**Comment by thanos1983** [comment]
> https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

On Ubuntu 21.04 the problem is fixed. It is related to the OS CA not to CRI-O.

But never the less the proposed workaround is simply run:

```bash
sudo apt-get update
sudo apt-get install ca-certificates
sudo apt-get upgrade
```

The problem is fixed after.

Hope this helps.

BR / Thanos

**Comment by iusetabs** [comment]
Thanks all for your input here. Will close this off now

**Comment by kluevandrew** [comment]
> ```shell
> sudo apt-get install ca-certificates
> ```

thx
